blazor/predefined-dialogs/action-buttons-dialog.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Customization of Action Buttons in Predefined Dialogs | Syncfusion
+description: Checkout and learn here all about customization of action buttons in Syncfusion Blazor Predefined Dialogs component and much more details.
+platform: Blazor
+control: Predefined Dialogs
+documentation: ug
+---
+
+# Customization of Action Buttons in Blazor Predefined Dialogs Component
```

Comment:
Add Customization.md file and combine action-buttons-dialog and close-button-dialog.md

Reply:
Combined action-buttons-dialog.md, close-button-dialog.md and customization-dialog-content.md files under Customization.md file sir.